### ISSUE TITLE
Unsupported symbol type 393318

### ISSUE THREAD

**Issue by dkorinets** [microsoft/navcontainerhelper#1678]
Hi Freddy, 

Have such issue when trying to run New-BcContainer
BcContainerHelper version 2.0.4

![image](https://user-images.githubusercontent.com/31474116/106744889-021bca00-6629-11eb-97f5-b8fb219db9c9.png)

The script: 
```
docker login $bcinsiderAzurecrIo -u $bcinsiderUser -p $bcinsiderPass
New-BcContainer `
    -accept_eula `
    -containerName $containerName `
    -credential $credential `
    -auth $auth `
    -artifactUrl $artifactUrl `
    -multitenant:$false `
    -includeTestToolkit `
    -includeTestLibrariesOnly `
    -assignPremiumPlan `
    -licenseFile $licenseFileUri `
    -memoryLimit 6G `
    -doNotExportObjectsToText `
    -updateHosts `
    -EnableTaskScheduler:$false
Setup-BcContainerTestUsers -containerName $containerName -Password $credential.Password -credential $credential

```
What could be the reason?

**Comment by freddydk** [comment]
Looks like setup-bccontainertestusers doesn't work with 18.0
Will look at that.

BTW - you don't need the docker login anymore - our images are public - artifacts are secured by a sas token, but that is not docker login

**Comment by freddydk** [comment]
Yeah, I will fix the bug, but in the meantime, you can run this as the last line:

```
Setup-BcContainerTestUsers -containerName $containerName -Password $credential.Password -credential $credential -createTestUsersAppUrl "https://businesscentralapps.azureedge.net/createtestusers/latest/apps.zip"
```

**Comment by freddydk** [comment]
Fix shipped in 2.0.5
